internal/cmd/kafka/command_acl_create.go

```diff
@@ -55,7 +55,7 @@ func (c *aclCommand) create(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	userIdMap, err := c.mapResourceIdToUserId()
+	users, userIdMap, err := c.mapResourceIdToUserId(nil)
```

Comment:
Would it make more sense to list the service accounts and users outside of this function, and pass in the combined list?

Reply:
+1 on that suggestion